en/admin/installation/installation.md

```diff
@@ -9,6 +9,12 @@ git clone https://github.com/wallabag/wallabag.git
 cd wallabag && make install
 ```
 
+If you want to use LDAP, replace last command with
+```bash
+cd wallabag && make install LDAP_ENABLED=true
+```
```

Comment:
Yes, but what about the installation when someone will implement a new feature? We'll have an install command with  `LDAP_ENABLED=true FEATURE_ENABLED=false ...`?